001_github_actions_importing_workflows/README.md

```diff
@@ -223,12 +223,13 @@ on demand.
 #### :arrow_right: Test the CUE workflow command
 
 With the modified `ci_tool.cue` file in place, check that the `regenerate`
-workflow command is available **from a shell sitting at the repo root**. For example:
+workflow command is available **from a shell sitting in the
+`internal/ci/github` directory**. For example:
 
 :computer: `terminal`
 ```sh
-cd $(git rev-parse --show-toplevel)            # make sure we're sitting at the repository root
-cue help cmd regenerate ./internal/ci/github   # the "./" prefix is required
+cd $(git rev-parse --show-toplevel)/internal/ci/github  # work around https://cuelang.org/issue/3462
+cue help cmd regenerate
```

Comment:
give me a bit of time to investigate this - this seems like a really unfortunate regression and I'd rather avoid the workaround entirely

Reply:
https://review.gerrithub.io/c/cue-lang/cue/+/1201709

Reply:
corredt, this was broken for v0.9.x and v0.10.x, and will be fixed in v0.11.0-alpha.2.